code/ZAS/Fire.dm

```diff
@@ -7,6 +7,7 @@ If it gains pressure too slowly, it may leak or just rupture instead of explodin
 */
 
 //#define FIREDBG
+#define MINIMUM_FUEL_VOLUME 0.0005 //Used to prevent leaving patches with astronomically tiny amounts of fuel
```

Comment:
This should be defined per reagent, in reagent define itself, so it can be extended to things like water.

Reply:
out of scope as discussed